node/src/components/consensus/highway_core/state/params.rs

```diff
@@ -72,6 +72,17 @@ impl Params {
         self.seed
     }
 
+    /// Returns the total reward for a finalized block.
+    pub fn block_reward(&self) -> u64 {
+        self.block_reward
+    }
+
+    /// Returns the reduced block reward that is paid out even if the heaviest summit does not
+    /// exceed half the total weight. This is at most `block_reward`.
+    pub fn reduced_block_reward(&self) -> u64 {
+        self.reduced_block_reward
+    }
```

Comment:
This is related to rewards v1, it should no longer be needed on feat-2.0.

Reply:
Removed in 080a969198464c157308f2b8dffd72450c0aece4